Archipelago.MultiClient.Net/BounceFeatures/DeathLink/DeathLinkService.cs

```diff
@@ -14,30 +14,44 @@ public class DeathLinkService
         public delegate void DeathLinkReceivedHandler(DeathLink deathLink);
         public event DeathLinkReceivedHandler OnDeathLinkReceived;
 
+        private DataStorageHelper logger;
+        private int slot;
+
         internal DeathLinkService(IArchipelagoSocketHelper socket)
         {
             this.socket = socket;
             socket.PacketReceived += OnPacketReceived;
+
+            logger = new DataStorageHelper(socket);
         }
 
-        void OnPacketReceived(ArchipelagoPacketBase packet)
+        private void OnPacketReceived(ArchipelagoPacketBase packet)
         {
-            if (packet.PacketType != ArchipelagoPacketType.Bounced)
+            switch (packet)
             {
-                return;
-            }
+                case ConnectedPacket connected:
+                    slot = connected.Slot;
+                    logger["DeathLinkReceived" + slot].Initialize(new string[0]);

```

Comment:
Should you add slot to the key here manually or just use a Slot scope?

Reply:
Was already changed to Scope.Slot in a later commit